CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ The point version is the next unused `v0.1.*` version, eg `v0.1.22` above. Creat
 
 ## Step 3. Build and deploy a new docker-cloud-action image, a new dagster.cloud.pex and update code references to docker
 
-A script does this work. **Note**: a virtual environment using Python3.8 is required to run the script.
+A script does this work. **Note**: a virtual environment using Python3.11 is required to run the script.
```

Comment:
why 3.11?
shouldn't we just pick one of the 3 defaults below?
maybe 310 because that's the default on internal?

Reply:
I did 3.11 because that's our default for development in general and because I changed a test case to depend on it

(3.10 is not the default on internal, 3.11 is)